README.md

```diff
@@ -65,6 +65,13 @@ Here we are adding mint.com to the password store, but more specifically to the
 ```
 $ passgo insert mney/mint.com
 ```
+#### passgo show group/pass-name
+Show is used to display a password in standard out. *Previously it was possible to display a password using `passgo group/pass-name` but this is no longer supported*.
```

Comment:
That's what my other comment was about. This one is about the missing blank line between lines 70 and 71.